config/examples/database.yml

```diff
@@ -35,7 +35,7 @@ development:
 
 test:
   <<: *default
-  url: <%= "#{ENV['DATABASE_URL']}#{ENV['TEST_ENV_NUMBER']}" %>
+  url: <%= "#{ENV['DATABASE_URL'].delete_suffix("_development")}test#{ENV['TEST_ENV_NUMBER']}" %>
```

Comment:
basically just set 
```
DATABASE_URL=postgresql://postgres:@localhost/3scale_system_development
```
and it will translate to `3scale_system_test` when running tests or `RAILS_ENV=test`